js/src/dom/manipulator.js

```diff
@@ -47,9 +47,13 @@ const Manipulator = {
       ...element.dataset
     }
 
-    Object.keys(attributes).forEach(key => {
-      attributes[key] = normalizeData(attributes[key])
-    })
+    Object.keys(attributes)
+      .filter(attributes => attributes.includes('bs'))
```

Comment:
I'm not 100% sure includes is the safest solution here. Aren't the attributes expected to always being with `bs`?

Reply:
Now using `startsWith` instead :)

Reply:
I'd go with a regex for now since we don't use `startsWith`/`endsWith` yet. I tried to switch to it in https://github.com/twbs/bootstrap/compare/master-xmr-js-modern-api but I'm hesitant. We'll tackle that later all at once.

Reply:
Actually, I only see one more case from a quick look that we could use `startsWith`, so leave it as is for now and we'll see later :)